src/modules/Home/components/Home.tsx

```diff
@@ -1,31 +1,113 @@
-import { Box, Paper } from '@mui/material';
-import { Avatar } from './Avatar';
-import { ResetText } from './ResetText';
-import { TextInfo } from './TextInfo';
-import { TextInput } from './TextInput';
+import {
+  Box,
+  Paper,
+  Typography,
+  Container,
+  Avatar,
+  Stack,
+} from '@mui/material';
+import { deepOrange } from '@mui/material/colors';
+import { getProjectList } from '@/data/projects';
+import { ProjectData } from '@/types/projects';
+import { useDeleteState, useProjectList, useAddProjectForm } from '../hooks';
+import { AddFormDialog } from './AddFormDialog';
+import { AddNewProject } from './AddNewProject';
+import { DeleteDialog } from './DeleteDialog';
+import { ProjectInfo } from './ProjectInfo';
 
 export function Home() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [projectsList, _] = useProjectList(getProjectList());
+
+  const [
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    deletableProjectId,
+    setDeletableProjectId,
+    deleteModalVisible,
+    setModalDeleteVisible,
+  ] = useDeleteState();
+
+  function clickDelete(projectId: string): void {
+    setDeletableProjectId(projectId);
+    setModalDeleteVisible(true);
+  }
+
+  const [
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    addFormData,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    setAddFormData,
+    addFormModalVisible,
+    setAddFormModalVisible,
+  ] = useAddProjectForm();
+
+  type FormHTMLElement = React.RefObject<HTMLFormElement>;
+
+  function onClickSubmit(addForm: FormHTMLElement): void {
+    addForm.current?.submit();
+    setAddFormModalVisible(false);
+  }
```

Comment:
Why did raise the submit handling to the parent component?
I consider it's the responsibility of `AddFormDialog` component since it has form and so it must submit the data of this form also